chart/values.schema.json

```diff
@@ -3161,12 +3161,12 @@
           "type": "boolean",
           "description": "Enabled defines if this option should be enabled."
         },
-        "translate": {
+        "patches": {
           "items": {
             "$ref": "#/$defs/TranslatePatch"
           },
           "type": "array",
-          "description": "Translate the patch according to the given patches."
+          "description": "Patches the resources according to the given patches."
```

Comment:
This is ambiguous, maybe keep "translate" here?

Reply:
How would I keep it here, given that this file gets generated off the Go struct?